src/CTA.FeatureDetection.AuthType/CompiledFeatures/WindowsAuthorizationRolesFeature.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Linq;
+using Codelyzer.Analysis;
+using Codelyzer.Analysis.Model;
+using CTA.FeatureDetection.Common.Extensions;
+
+namespace CTA.FeatureDetection.AuthType.CompiledFeatures
+{
+    public class WindowsAuthorizationRolesFeature : WebConfigFeature
+    {
+        private readonly string _path = $"{Constants.ConfigurationElement}/{Constants.SystemWebElement}/{Constants.AuthorizationElement}/{Constants.AllowElement}";
```

Comment:
We are computing local path variables often can we add these as part of the constants class instead to be put together from its own members?

Reply:
Will do- thanks!